digicampipe/io/generate_slow_containers.py

```diff
@@ -0,0 +1,203 @@
+        print('    return event', file=file)
+    print('\n', file=file)
+
+
+def generate_fill_slow(filenames, file=sys.stdout):
+    print('def fill_slow(class_name, event, hdu, slow_event):', file=file)
+    first = True
+    for path_file in filenames:
+        class_name = get_class_name(path_file)
+        if first:
+            print('    if class_name == "%s":' % class_name, file=file)
+            first = False
+        else:
+            print('    elif class_name == "%s":' % class_name, file=file)
+        print('        event = fill_%s(event, hdu, slow_event)'
+              % class_name, file=file)
+    print('    else:', file=file)
+    print('        print("ERROR in fill_slow():'
+          ' class %s not known." %class_name)', file=file)
+    print('        print("Try to regenerate slow data containers ?")',
+          file=file)
+    print('    return event', file=file)
+    print('\n', file=file)
+
+
+def generate_header(file=sys.stdout):
+    print('# WARNING ! this file has been generated by '
+          'generate_slow_containers.py script.', file=file)
+    print('# Do NOT edit this file manually as the changes would be '
+          'lost when the script is re-run.', file=file)
+    print('# Change instead the generate_slow_containers.py script to '
+          'create the changes you want.\n', file=file)
+    print('from numpy import ndarray, uint8, int64, int32, float32, float64'
+          , file=file)
+    print('', file=file)
+    print('from astropy import units as u', file=file)
+    print('from ctapipe.core import Container', file=file)
+    print('try:', file=file)
+    print('    from ctapipe.core import Field', file=file)
+    print('except ImportError:', file=file)
+    print('    from ctapipe.core import Item as Field', file=file)
+    print('\n', file=file)
+
+
+def generate_slow_container(filenames, file=sys.stdout):
+    print('class SlowDataContainer(Container):', file=file)
+    for path_file in filenames:
+        class_name = get_class_name(path_file)
+        print('    %s = Field(%sContainer(), "%s")'
+              %(class_name.lower(), class_name, class_name),
+              file=file)
+    print('\n', file=file)
+
+
+def generate_class_list(filenames, file=sys.stdout):
+    print('__all__ = [', file=file)
+    for path_file in filenames:
+        class_name = get_class_name(path_file)
+        print('    "%sContainer",' % class_name, file=file)
+        print('    "fill_%s",' % class_name, file=file)
+    print('    "SlowDataContainer",', file=file)
+    print('    "fill_slow",', file=file)
+    print(']\n\n', file=file)
+
+
+if __name__ == '__main__':
+    filenames = glob(
+        pkg_resources.resource_filename(
+            'digicampipe',
+            'tests/resources/slow/*.fits'
+        )
+    )
+
+    # get list of files to treat
+    # (to not treat classes more than once).
+    files_to_process = []
+    classes_to_process = []
+    for path_file in filenames:
```

Comment:
It would be nice to have this part of the code to be handled in one function/class. Avoiding that the user does it in the wrong order. someting like:

```python
def write_slow_control_data_container(example_slow_file, file_to_write):
    ...
```